### PR TITLE
Command Handler Added

### PR DESCRIPTION
**Pull request by nerdthatnoonelikes**
This pull request added a command handler so it would be easier to manage multiple commands and for ease of readability.

### CODE REVIEW

**Review by carlank — CHANGES_REQUESTED** [review]
Overall, totally great! I made a couple of small remarks, and in general I'm not a fan of the circular imports (maybe extend the Client as an actual new class, adding a Bot property, and either add the commands Collection there and make the commands instances of a Command class that is constructed with a Client, or just have the commands Collection be separate and pull the bot from the client? I like the former, personally. Feels more in line with how we're doing stuff in general. )

When I get back to my computer, I'll set up eslint as part of the commit flow, so I'm not  too worried about style inconsistencies right now. 

@Stratadox thoughts?

**Comment by carlank** [comment]
Oh, I can implement these changes myself too, or we can merge this if @Stratadox or anyone else approves and just do the fixes as another pr, just will take me a day or two. Doing this from my phone.

File: index.js [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+const commandFiles = fs.readdirSync('./src/commands').filter(file => file.endsWith('.js'));
+
+for (const file of commandFiles) {
+  const command = require(`./src/commands/${file}`)
+
+  client.commands.set(command.name, command)
+}
+
 const bot = new Bot();
+module.exports = bot;
```
Comment (Stratadox) [inline index.js:19]: I'm not a big fan of making the bot instance globally available tbh.

Comment (carlank) [reply index.js:19]: I'm in agreement. Did you see my other comment about a potential way to structure this instead?

Comment (Stratadox) [reply index.js:19]: Just read up on it. Sounds good :) I do like the idea of separating the commands into specific handlers.. Might improve extensibility and testability.

**Comment by nerdthatnoonelikes** [comment]
Exporting the bot instance is better so this way it can be accessed from the individual command files.

**Comment by carlank** [comment]
I think one of the solutions I mentioned in the earlier comment might be better than exporting bot like that. I'll be back at a computer in a few hours and I can throw together an example of what I mean!

**Pull request merged**